components/admin/Footer.tsx

```diff
@@ -0,0 +1,25 @@
+import { Portal, Box } from '@chakra-ui/react'; // Chakra UI
+import { ReactNode } from 'react'; // React
+
+type ProcessingTasksFooterProps = {
+  readonly children: ReactNode;
+};
+
+export default function ProcessingTasksFooter({ children }: ProcessingTasksFooterProps) {
```

Comment:
```suggestion
export default function Footer({ children }: ProcessingTasksFooterProps) {
```
We will use this component as a generic footer wrapper